lib/lambda/footprint-app.ts

```diff
@@ -3,8 +3,8 @@ const awsServerlessExpressMiddleware = require('aws-serverless-express/middlewar
 const bodyParser = require('body-parser')
 import express from 'express'
 
-const TABLE_NAME = process.env.TABLE_NAME || "";
-const MOCK = process.env.mock || false
+const TABLE_NAME = process.env.TABLE_NAME || ''
+const MOCK = process.env.AWS_EXECUTION_ENV === 'local-mock' || false
```

Comment:
ここlocalstackの環境上でも動くようにしたいのでにしたいです！

```suggestion
const MOCK = process.env.LOCALSTACK_HOSTNAME === 'localhost' || false
```

Reply:
@ayuki-joto 
了解です。テストコードの方の環境変数設定も変更します。